draft-ietf-quic-transport.md

```diff
@@ -2133,25 +2133,25 @@ These states SHOULD persist for three times the current Retransmission Timeout
 
 An endpoint enters a closing period after initiating an immediate close
 ({{immediate-close}}).  While closing, an endpoint MUST NOT send packets unless
-they contain a CONNECTION_CLOSE or APPLICATION_CLOSE frame (see
-{{immediate-close}} for details).  An endpoint retains only enough information
-to generate a packet containing a closing frame and to identify packets as
-belonging to the connection.  The connection ID and QUIC version is sufficient
-information to identify packets for a closing connection; an endpoint can
-discard all other connection state.  An endpoint MAY retain packet protection
-keys for incoming packets to allow it to read and process a closing frame.
+they contain a CONNECTION_CLOSE frame (see {{immediate-close}} for details).  An
+endpoint retains only enough information to generate a packet containing a
+CONNECTION_CLOSE frame and to identify packets as belonging to the connection.
+The connection ID and QUIC version is sufficient information to identify packets
+for a closing connection; an endpoint can discard all other connection state.
+An endpoint MAY retain packet protection keys for incoming packets to allow it
+to read and process a CONNECTION_CLOSE frame.
```

Comment:
This is outside the semantic changes of the PR, but I wonder if connection close retransmission could suffer from lack of address validation. Amplification attacks would only be an issue if the close packet is large. One could imagine a valid connection triggering a large close packet that is not acknowledged followed by a spoofed packets that redirects retransmission. I don't think this is practical, but perhaps some clarification on the subject would be helpful. For example recommending sending small close packets if not verified.

Reply:
I think the Immediate close text below addresses your concern?

Reply:
Not really, though I could easily miss something:

 "During the closing period, an endpoint that sends a CONNECTION_CLOSE
 frame SHOULD respond to any packet that it receives with another packet
 containing a CONNECTION_CLOSE frame. "

If this is done without verying the input packet that triggers the next close frame because it is a stateless shrink-wrapped close packet triggered by CID, there is a potential for abuse, I believe.